contrib/dcr_tmux_simnet_setup.sh

```diff
@@ -131,7 +131,10 @@ tmux split-window -v
 tmux select-pane -t 0
 tmux resize-pane -D 15
 tmux send-keys "cd ${NODES_ROOT}/${PRIMARY_WALLET_NAME}" C-m
-tmux send-keys "dcrwallet -C ../wallet.conf --create <../wallet.answers; tmux wait-for -S ${PRIMARY_WALLET_NAME}" C-m
+tmux send-keys "dcrwallet -C ../wallet.conf --create <<EOF
+${WALLET_CREATE_CONFIG}
+EOF
```

Comment:
if it's all saved in a variable why not just `echo $WALLET_CREATE_CONFIG | dcrwallet`?

Reply:
I tried that and it didn't work properly with the multiline input.

Reply:
It ends up with
```
$ echo ynyb280922d2cffda44648346412c5ec97f429938105003730414f10b01e1402eac | dcrwallet -C ../wallet.conf --create; tmux wait-for -S wallet1
... Use the existing configured private passphrase for wallet encryption? (n/no/y/yes) [no]: Unable to create wallet: EOF
```

Reply:
that's not the full input needed though..

Reply:
Correct.

Reply:
Should work:

```
$ foo="a
> b
> c
> "
$ echo "$foo" | cat
a
b
c

$ 
```

Reply:
Yeah, I thought it should too, and I had also confirmed it works in the raw shell, but it didn't work in the script.

I figured out why though now.  It's because `tmux send-keys` is quoted and the var has to be quoted too, so those quotes need to be escaped since you can't wrap the overall `tmux send-keys` in single quotes as that disabled variable expansion.

Updated.